### PR TITLE
chore: use queued build status for queued builds

### PR DESCRIPTION
**Pull request by shreddedbacon**
# Checklist

- [ ] Affected Issues have been mentioned in the Closing issues section
- [ ] Documentation has been written/updated
- [ ] PR title is ready for changelog and subsystem label(s) applied

Now that https://github.com/uselagoon/lagoon/pull/3351 has been merged, this changes the controller to send the queued status for queued builds.

### CODE REVIEW

**Comment by shreddedbacon** [comment]
This has a requirement of Lagoon v2.12.0 to work, so will try to coordinate this some how

File: internal/helpers/helpers.go [hunk 1]
```diff
@@ -219,6 +221,52 @@ func VariableExists(vars *[]LagoonEnvironmentVariable, name, value string) bool
 	return exists
 }
 
+// GetLagoonVariable returns a given environment variable
+func GetLagoonVariable(name string, scope []string, variables []LagoonEnvironmentVariable) (*LagoonEnvironmentVariable, error) {
+	for _, v := range variables {
+		scoped := true
+		if scope != nil {
+			scoped = false
```
Comment (bomoko) [inline internal/helpers/helpers.go:229]: I think you could replace with all just with
scoped = constainsStr(scope, v.Scope)

Comment (bomoko) [reply internal/helpers/helpers.go:229]: lines 229-232, I mean

**Review by bomoko — APPROVED** [review]
LGTM, just added one more comment, but happy happy

File: internal/helpers/helpers.go (hunk 1 above)
Comment (shreddedbacon) [reply internal/helpers/helpers.go]: :see_no_evil: :force:

**Pull request merged**